packages/dds/tree/src/simple-tree/api/schemaFactoryAlpha.ts

```diff
@@ -3,37 +3,34 @@
  * Licensed under the MIT License.
  */
 
-import type { ScopedSchemaName, InsertableObjectFromSchemaRecord } from "../internalTypes.js";
-import {
-	SchemaFactory,
-	type SchemaFactoryObjectOptions,
-	type TreeNodeSchemaClass,
-	type NodeKind,
-	type Unenforced,
-	type ImplicitFieldSchema,
-} from "../simple-tree/index.js";
-// eslint-disable-next-line import/no-internal-modules
-import { defaultSchemaFactoryObjectOptions } from "../simple-tree/api/schemaFactory.js";
-// eslint-disable-next-line import/no-internal-modules
-import { type TreeObjectNode, objectSchema } from "../simple-tree/objectNode.js";
-import type { RestrictiveStringRecord } from "../util/index.js";
+import type {
+	ScopedSchemaName,
+	InsertableObjectFromSchemaRecord,
+	TreeObjectNodeUnsafe,
+	InsertableObjectFromSchemaRecordUnsafe,
+} from "../../internalTypes.js";
+import { SchemaFactory, type SchemaFactoryObjectOptions } from "./schemaFactory.js";
+import type { ImplicitFieldSchema } from "../schemaTypes.js";
+import { defaultSchemaFactoryObjectOptions } from "./schemaFactory.js";
+import { type TreeObjectNode, objectSchema } from "../objectNode.js";
+import type { RestrictiveStringRecord } from "../../util/index.js";
+import type { NodeKind, TreeNodeSchemaClass } from "../core/index.js";
+import type { Unenforced } from "./typesUnsafe.js";
 
 /**
- * Copy of {@link SchemaFactory} with additional alpha APIs.
+ * {@link SchemaFactory} with additional alpha APIs.
  *
+ * @alpha
  * @privateRemarks
- * Not currently exported to the public API surface as doing so produces errors in API-extractor.
  *
- * Can be removed once additional object node features are deemed stable and on the base class.
+ * Defining this class separately allows correct API isolation of alpha APIs (method signatures) on SchemaFactory.
+ * Prefer defining duplicate private methods in `SchemaFactoryAlpha` rather than increasing their exposure, as `protected`
```

Comment:
Nit: it took me a couple of re-reads to understand what you meant here. Might be worth attempting to reword.